test_conformance/common/vulkan_wrapper/vulkan_wrapper.cpp

```diff
@@ -174,18 +218,57 @@ VulkanInstance::VulkanInstance(): m_vkInstance(VK_NULL_HANDLE)
 
     VkInstanceCreateInfo vkInstanceCreateInfo = {};
     vkInstanceCreateInfo.sType = VK_STRUCTURE_TYPE_INSTANCE_CREATE_INFO;
-    vkInstanceCreateInfo.pNext = NULL;
     vkInstanceCreateInfo.flags = 0;
     vkInstanceCreateInfo.pApplicationInfo = &vkApplicationInfo;
-    vkInstanceCreateInfo.enabledLayerCount = 0;
     vkInstanceCreateInfo.ppEnabledLayerNames = NULL;
     vkInstanceCreateInfo.enabledExtensionCount =
         (uint32_t)enabledExtensionNameList.size();
     vkInstanceCreateInfo.ppEnabledExtensionNames =
         enabledExtensionNameList.data();
+    vkInstanceCreateInfo.enabledLayerCount = 0;
+    vkInstanceCreateInfo.pNext = NULL;
+
+    VkDebugUtilsMessengerCreateInfoEXT debugCreateInfo{};
+    if (m_useValidationLayers)
+    {
+        vkInstanceCreateInfo.enabledLayerCount =
+            static_cast<uint32_t>(m_validationLayers.size());
+        vkInstanceCreateInfo.ppEnabledLayerNames = m_validationLayers.data();
+
+        debugCreateInfo.messageType =
+            VK_DEBUG_UTILS_MESSAGE_TYPE_GENERAL_BIT_EXT
+            | VK_DEBUG_UTILS_MESSAGE_TYPE_VALIDATION_BIT_EXT
+            | VK_DEBUG_UTILS_MESSAGE_TYPE_PERFORMANCE_BIT_EXT;
+
+        // VK_DEBUG_UTILS_MESSAGE_SEVERITY_VERBOSE_BIT_EXT
+        // VK_DEBUG_UTILS_MESSAGE_SEVERITY_INFO_BIT_EXT
+        // VK_DEBUG_UTILS_MESSAGE_SEVERITY_WARNING_BIT_EXT
+        // VK_DEBUG_UTILS_MESSAGE_SEVERITY_ERROR_BIT_EXT
+        debugCreateInfo.messageSeverity |=
+            VK_DEBUG_UTILS_MESSAGE_SEVERITY_ERROR_BIT_EXT;
+        debugCreateInfo.sType =
+            VK_STRUCTURE_TYPE_DEBUG_UTILS_MESSENGER_CREATE_INFO_EXT;
+        debugCreateInfo.messageSeverity =
+            VK_DEBUG_UTILS_MESSAGE_SEVERITY_WARNING_BIT_EXT
+            | VK_DEBUG_UTILS_MESSAGE_SEVERITY_ERROR_BIT_EXT;
```

Comment:
We should only set `debugCreateInfo.messageSeverity` once.

Reply:
Done, thanks!